### PR TITLE
Enable useNativeDriver to improve performance

### PR DESCRIPTION
**Pull request by weswes**
Issue related: https://github.com/atlanteh/react-native-slot-machine/issues/5

Changed top layout animation to translateY and added useNativeDriver.
Default to true as it's way better.

### CODE REVIEW

**Comment by atlanteh** [comment]
Thx for the PR! Can you please change the default useNativeDriver prop (and docs) to false, as to not break compatibility?

File: index.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ export default class SlotMachine extends Component {
             initialAnimation: true,
             styles: {},
             renderTextContent: (currentChar) => currentChar,
+	    useNativeDriver: true,
```
Comment (atlanteh) [inline index.js]: Please change to false, and align it to other columns

File: README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ The following props can be used to modify the slot machine's style and/or behavi
 |__`renderTextContent`__|_Function_|Optional|`(char, index, range) => char`|Allows replacing the inner content of the Text element
 |__`renderContent`__|_Function_|Optional|`(char, index, range) => char`|Allows replacing the entire Text element with your own implementation
 |__`styles`__|_Object_|Optional|`{}`|Allows overriding each of the inner components (container, slotWrapper, slotInner, innerBorder, outerBorder, overlay, text)
+|__`useNativeDriver`__|_Boolean_|Optional|`true`|Enable use of NativeDriver on Animation. See https://facebook.github.io/react-native/docs/animations.html#using-the-native-driver
```
Comment (atlanteh) [inline README.md]: Please change default to false

File: index.js [hunk 3]
```diff
@@ -109,7 +110,7 @@ export default class SlotMachine extends Component {
             const newAnimations = paddedStr.split('').map((char, i) => {
                 const index = range.indexOf(char);
                 const animationValue = -1 * (index) * newProps.height;
-                return Animated.timing(this.state.values[i], {toValue: animationValue, duration, easing});
+                return Animated.timing(this.state.values[i], {toValue: animationValue, duration, easing, useNativeDriver: this.props.useNativeDriver});
```
Comment (atlanteh) [inline index.js]: take it from newProps, not this.props. Also, you can declare useNativeDriver on line 104, and just use it here.

File: index.js [hunk 4]
```diff
@@ -191,7 +192,7 @@ export default class SlotMachine extends Component {
 
         const animations = values.map((value, i) => {
             const animationDuration = duration - (values.length - 1 - i) * slotInterval;
-            return Animated.timing(value, {toValue: 0, duration: animationDuration, easing});
+            return Animated.timing(value, {toValue: 0, duration: animationDuration, easing, useNativeDriver: this.props.useNativeDriver});
```
Comment (atlanteh) [inline index.js]: declare variable in line 190 and use here.

**Review by atlanteh — CHANGES_REQUESTED** [review]
Please fix the above and we can merge it :)
Thx!

**Comment by weswes** [comment]
Done. 
I hesitated to put default to false as it's fully backward compatible and it's better with true but you decide ;)

**Review by atlanteh — APPROVED** [review]

**Pull request merged**